### PR TITLE
Feature/icky- 180 & 181

### PR DESCRIPTION
**Pull request by kibo-geetanshuchhabra**

### CODE REVIEW

File: framework/kibocommerce/api/endpoints/wishlist/add-item.ts [hunk 1]
```diff
@@ -1,25 +1,77 @@
 import getCustomerWishlist from '../../operations/get-customer-wishlist'
-// import { parseWishlistItem } from '../../utils/parse-item'
-// import getCustomerId from '../../utils/get-customer-id'
+import getCustomerId from '../../utils/get-customer-id'
 import type { WishlistEndpoint } from '.'
+import { normalizeWishlistItem } from '@framework/lib/normalize'
+import { getProductQuery } from '@framework/api/queries/get-product-query'
+import addItemToWishlist from '../../mutations/addItemToWishlist-mutation'
```
Comment (kibo-kevinwatts) [inline framework/kibocommerce/api/endpoints/wishlist/add-item.ts:6]: Rename `addItemToWishlist` to `addItemToWishlistMutation`

File: framework/kibocommerce/api/endpoints/wishlist/remove-item.ts [hunk 2]
```diff
@@ -1,7 +1,9 @@
 // import type { Wishlist } from '@commerce/types/wishlist'
 import getCustomerWishlist from '../../operations/get-customer-wishlist'
-// import getCustomerId from '../../utils/get-customer-id'
+import getCustomerId from '../../utils/get-customer-id'
 import type { WishlistEndpoint } from '.'
+import { normalizeWishlistItem } from '@framework/lib/normalize'
+import removeItemFromWishlist from '../../mutations/removeItemFromWishlist-mutation'
```
Comment (kibo-kevinwatts) [inline framework/kibocommerce/api/endpoints/wishlist/remove-item.ts:6]: Rename `removeItemFromWishlist` to `removeItemFromWishlistMutation`

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]

File: framework/kibocommerce/api/endpoints/wishlist/add-item.ts (hunk 1 above)
Comment (kibo-geetanshuchhabra) [reply framework/kibocommerce/api/endpoints/wishlist/add-item.ts]: Resolved

File: framework/kibocommerce/api/endpoints/wishlist/remove-item.ts (hunk 2 above)
Comment (kibo-geetanshuchhabra) [reply framework/kibocommerce/api/endpoints/wishlist/remove-item.ts]: Resolved

**Pull request merged**